app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/ImageRegionSelectionInteractionViewModel.kt

```diff
@@ -49,25 +54,58 @@ class ImageRegionSelectionInteractionViewModel private constructor(
       object : Observable.OnPropertyChangedCallback() {
         override fun onPropertyChanged(sender: Observable, propertyId: Int) {
           errorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
-            pendingAnswerError = null,
-            inputAnswerAvailable = answerText.isNotEmpty()
+            pendingAnswerError = pendingAnswerError, // Allow blank answer submission.
+            inputAnswerAvailable = true
           )
         }
       }
     isAnswerAvailable.addOnPropertyChangedCallback(callback)
+    errorMessage.addOnPropertyChangedCallback(callback)
+
+    // Initializing with default values so that submit button is enabled by default.
+    errorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
+      pendingAnswerError = null,
+      inputAnswerAvailable = true
+    )
   }
 
   override fun onClickableAreaTouched(region: RegionClickedEvent) {
+
     when (region) {
       is DefaultRegionClickedEvent -> {
         answerText = ""
         isAnswerAvailable.set(false)
+        isDefaultRegionClicked = true
       }
       is NamedRegionClickedEvent -> {
         answerText = region.regionLabel
         isAnswerAvailable.set(true)
       }
     }
+    checkPendingAnswerError(AnswerErrorCategory.REAL_TIME)
+  }
+
+  /** It checks the pending error for the current image region input, and correspondingly updates the error string based on the specified error category. */
+  override fun checkPendingAnswerError(category: AnswerErrorCategory): String? {
+    when (category) {
+      AnswerErrorCategory.REAL_TIME -> {
+        pendingAnswerError = null
+      }
+
+      AnswerErrorCategory.SUBMIT_TIME -> {
+        if (answerText.isNotEmpty() || isDefaultRegionClicked) {
```

Comment:
Should this not be `&&`? Submit time Error is only null if there is no interaction whatsoever. Otherwise, why do we need to have the `isDefaultRegionClicked` error?

Reply:
`isDefaultRegionClicked  `is required when user select DefaultRegion and click on submit button, if we eliminate `isDefaultRegionClicked`, it will show error message, instead of "This doesn't seems like correct choice. Try again". And same with `&&`.